src/lib.rs

```diff
@@ -76,15 +76,16 @@ mod test {
         "#
     );
 
-    test!(SYNTAX, runner,
+    test_inline!(SYNTAX, runner,
```

Comment:
Looks like in `@swc/core@^1.4.0` const enum handled by swc, will compile into var. Should we remove this test

https://play.swc.rs/?version=1.4.12&code=H4sIAAAAAAAAA0vOzysuUUjNK81VcMvPV6jmUgACRx0w5cRVywUAZWdi5iAAAAA%3D&config=H4sIAAAAAAAAA2WPSw7DIAxE9zkF8rrbbnqHHgJRJyLiJ%2BxIRVHuXpxA0qobhJ9nPPY6KAUzGXiotX5rkXQmzGddCZXA%2Bl0JcElIJtvEcOtdJmmN2hHuaDs6wDpPyOJCujc5uBgJu7wxb4Mdy3egiT5lJPoVilSHyeF%2F3P5cg04TWHrG17JbOC8H8520g%2BUoWbOm%2BhhmAhk4bB9kMbooGgEAAA%3D%3D

Reply:
I think adding `export` keyword to this test is fine. Just keep it.